src/pages/workspaces/workspace/analysis/tool-utils.ts

```diff
@@ -83,6 +92,25 @@ export const tools: Record<ToolLabel, Tool> = {
   spark: { label: 'spark' }
 }
 
+//TODO: Should the value be a list?
```

Comment:
Are you still trying to sort this out?

Reply:
Was hoping for feedback, but I think it's fine as-is